packages/rolldown/src/binding.js

```diff
@@ -60,7 +60,13 @@ const isMuslFromChildProcess = () => {
 }
 
 function requireNative() {
-  if (process.platform === 'android') {
+  if (process.env.NAPI_RS_NATIVE_LIBRARY_PATH) {
```

Comment:
What is this used for? This did not seem to be auto generated, can you add some doc about this?

Reply:
It's auto generated.

Reply:
ok, i forgot to reinstall.